q2_metadata/plugin_setup.py

```diff
@@ -114,14 +114,20 @@ def _2(ff: ArtificialGroupingFormat) -> (qiime2.Metadata):
     parameters={'metadata': MetadataColumn[Categorical],
                 'n_columns': Int,
                 'column_name_prefix': Str,
-                'column_value_prefix': Str},
+                'column_value_prefix': Str,
+                'encode_sample_size': Bool
+                },
     parameter_descriptions={
         'metadata': ('Categorical metadata column to shuffle.'),
         'n_columns': 'The number of shuffled metadata columns to create.',
         'column_name_prefix': ('Prefix to use in naming the shuffled '
                                'metadata columns.'),
         'column_value_prefix': ('Prefix to use in naming the values in the '
-                                'shuffled metadata columns.')},
+                                'shuffled metadata columns.'),
+        'encode_sample_size': ('If true, the sample size of each metadata'
+                               'group will be appended to the shuffled'
+                               'metadata column value names.'),
```

Comment:
I reworded this slightly so that it was a bit more clear that we are referring to groups within a given metadata column vs. metadata groups (which isn't language our users would be familiar with)